src/test/java/org/springframework/data/couchbase/repository/ReactiveCouchbaseRepositoryKeyValueIntegrationTests.java

```diff
@@ -64,13 +66,13 @@ void saveAndFindById() {
 	}
 
 	@Test
-	void findBySimplePropertyAudited() {
+	void findByIdAudited() {
 		Airport vie = null;
 		try {
 			vie = new Airport("airports::vie", "vie", "low2");
 			Airport saved = airportRepository.save(vie).block();
-			List<Airport> airports1 = airportRepository.findAllByIata("vie").collectList().block();
-			assertEquals(saved, airports1.get(0));
+			Airport airport1 = airportRepository.findById(saved.getId()).block();
+			assertEquals(airport1, saved);
```

Comment:
This was using a query which requires an index, but if you run this test by itself, there won't be an index.  Switch to use kv.